.github/workflows/ci.yaml

```diff
@@ -10,10 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-        with:
-          persist-credentials: false
-          token: ${{ secrets.GH_TOKEN }}
+        uses: actions/checkout@v3
```

Comment:
if you rebased, this shouldn't show in your PR... please rebase, don't just merge.
```sh
git fetch -ap
git rebase origin/main
```

Reply:
Not sure what happened, but now it is properly rebased.